File: C/bee1036.c
#include <stdio.h>
#include <math.h>

int main(){
    double a,b,c,delta,r1,r2, raizDelta;
    scanf("%lf %lf %lf", &a,&b,&c);
    delta = ((b*b)-4*a*c);
    raizDelta = sqrt(delta);
    if (delta < 0 || a == 0)
    {
        printf("Impossivel calcular\n");
    } else {
        r1 = (-b+raizDelta) / (2*a);
        r2 = (-b-raizDelta) / (2*a);
        printf("R1 = %.5f\nR2 = %.5f\n", r1, r2);
    }
}

File: js/bee1036.js
var input = require('fs').readFileSync('js/stdin', 'utf8')

var [a,b,c] = input.split(' ').map(Number)
var delta = ((b**2)-4*a*c)
var Rdelta = Math.sqrt(delta)
var r1 = (-b+Rdelta)/(2*a)
var r2 = (-b-Rdelta)/(2*a)

if ((a === 0) || (delta < 0)){
    console.log('Impossivel calcular')
} else {
    console.log("R1 = " + r1.toFixed(5))
    console.log("R2 = " + r2.toFixed(5))
}